server/meshmodel/kubernetes/relationships/hierarchical_inventory.json

```diff
@@ -1077,6 +1077,40 @@
           "to": []
         }
       }
+    },
+    {
+      "allow": {
+        "from":[
+          {
+            "kind": "StorageClass",
+            "model": "kubernetes",
+            "patch": {
+              "patchStrategy": "replace",
+              "mutatorRef": [
+                ["name"]
+              ],
+              "description": "A StorageClass in Kubernetes defines the type and properties of storage that can be dynamically provisioned for persistent volumes."
```

Comment:
Please add a description of how one component modifies or is modified by the other.